gapic/src/main/com/google/gapid/models/Perfetto.java

```diff
@@ -243,6 +253,15 @@ public GpuInfo getGpu() {
         return gpu;
       }
 
+      public SfEvents getSfEvents() {
+        return sfEvents;
+      }
+
+      public Builder setSfEvents(SfEvents sfEvents) {
+        this.sfEvents = sfEvents;
+        return this;
```

Comment:
I know this exists in the other set* methods but, do you know why do we need to return the instance after setting the value??

Reply:
It is just integrated like that in the function to avoid an extra return line in the caller of this function I feel. Take a look at SfEvents.java line 52 for example. The return type is also Builder and the parameter Data is also of Builder type. Instead of writing a separate return Data line, I think its just integrated here.